css-three-column-layout/styles.css

```diff
@@ -0,0 +1,46 @@
+.container {
+  background-color: rgb(26 150 218);
+  color: white;
+  text-align: center;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  border-width: 40px;
+  padding: 40px 50px;
```

Comment:
All of these should be applied to another class separate from the `container` class. In a layout class system, the only valid properties on the `container` class are `margin` and `max-width`